### DISCUSSION TITLE
vercel dev error on page load

### DISCUSSION THREAD

**Discussion by mjedmonds** [vercel/community#2524] [Help] [answered]
### Summary

I'm trying to use `vercel dev` for the [python flask hello world](https://vercel.com/templates/python/flask-hello-world). However, I'm getting the following error when I load the page:

```
$ vercel dev
Vercel CLI 29.3.6
> Ready! Available at http://localhost:3000
> Building @vercel/python@latest:api/index.py
Installing required dependencies...
> Built @vercel/python@latest:api/index.py [2s]
Error: spawn /Users/mark/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
Error: An unexpected error occurred!
Error: spawn /Users/mark/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

Any ideas what could be causing this issue? I've tried using different versions of Node and still get the same error. It seems to be a permission issue, but I'm not sure where / why. The same app deploys on Vercel and runs successfully live.

Thanks in advance!

**Comment by mjedmonds** [comment]
I fixed this with 

```
chmod 777 /Users/mark/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap
```

but that is a suboptimal solution. What should the ownership / permissions be on this?

**Comment by loopj** [comment, marked as answer]
I'm seeing the exact same issue on my M1 Macbook. Running `chmod +x ~/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap` fixed it for me.

**Comment by celius1997** [comment]
I was experiencing the exact same issue. Running `chmod 777 /Users/celiafl/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap` did the job. thank you very much!

**Comment by samrocksc** [comment]
Just wanted to write in this also fixes it with the same issue in rust as well.
